sql/core/src/test/resources/sql-tests/results/pgSQL/int2.sql.out

```diff
@@ -0,0 +1,399 @@
+-- Automatically generated by SQLQueryTestSuite
+-- Number of queries: 38
+
+
+-- !query 0
+CREATE TABLE INT2_TBL(f1 smallint) USING parquet
+-- !query 0 schema
+struct<>
+-- !query 0 output
+
+
+
+-- !query 1
+INSERT INTO INT2_TBL VALUES (trim('0   '))
+-- !query 1 schema
+struct<>
+-- !query 1 output
+
+
+
+-- !query 2
+INSERT INTO INT2_TBL VALUES (trim('  1234 '))
+-- !query 2 schema
+struct<>
+-- !query 2 output
+
+
+
+-- !query 3
+INSERT INTO INT2_TBL VALUES (trim('    -1234'))
+-- !query 3 schema
+struct<>
+-- !query 3 output
+
+
+
+-- !query 4
+INSERT INTO INT2_TBL VALUES ('32767')
+-- !query 4 schema
+struct<>
+-- !query 4 output
+
+
+
+-- !query 5
+INSERT INTO INT2_TBL VALUES ('-32767')
+-- !query 5 schema
+struct<>
+-- !query 5 output
+
+
+
+-- !query 6
+SELECT '' AS five, * FROM INT2_TBL
+-- !query 6 schema
+struct<five:string,f1:smallint>
+-- !query 6 output
+-1234
```

Comment:
Ur, is this our test framework behavior? The first column seems to be gone.

Reply:
Yes. This is the behavior if replace `''` to `'#'`:
![image](https://user-images.githubusercontent.com/5399861/60861168-0a910080-a24c-11e9-91b4-0efbd97d3dbe.png)